platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1372,10 +1372,20 @@ private Response continueLogout(
                 logoutState.getNameId(),
                 SystemBaseUrl.constructUrl(IDP_LOGIN, true),
                 logoutState.getSessionIndexes());
+
+        entityServiceInfo =
+            getServiceProvidersMap().get(entityId).getLogoutService(incomingBinding);
+
+        logoutRequest.setDestination(
+            getServiceProvidersMap()
+                .get(entityId)
+                .getLogoutService(entityServiceInfo.getBinding())
+                .getUrl());
```

Comment:
❓ Is there a reason you didn't just call `entityServiceInfo.getUrl()`?

Reply:
Nope, I was just copying the way we were doing it below. I'll change both 👍